src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1012,6 +1013,20 @@ public static void onEmptyLeftClick(EntityPlayer player, ItemStack stack)
         MinecraftForge.EVENT_BUS.post(new PlayerInteractEvent.LeftClickEmpty(player, stack));
     }
 
+    public static CriticalHitEvent getCriticalHit(EntityPlayer player, Entity target, boolean vanillaCritical, float vanillaDefault)
```

Comment:
please change everything named `vanillaDefault` to `damageModifier`, it is more clear